server/src/test/java/keywhiz/test/ServiceContext.java

```diff
@@ -0,0 +1,71 @@
+package keywhiz.test;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Throwables;
+import com.google.common.io.Resources;
+import io.dropwizard.configuration.ConfigurationException;
+import io.dropwizard.configuration.YamlConfigurationFactory;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.setup.Environment;
+import java.io.File;
+import java.io.IOException;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import keywhiz.KeywhizConfig;
+import keywhiz.KeywhizService;
+
+public class ServiceContext {
+  private final KeywhizService service;
+  private final Bootstrap<KeywhizConfig> bootstrap;
+  private final KeywhizConfig config;
+  private final Environment environment;
```

Comment:
For testing, these four classes are often required and their construction is intertwined. Encapsulating them all together in a single class made creation and consumption much easier and reduced boilerplate scattered about the codebase.